content/domain.ttl

```diff
@@ -2,4 +2,8 @@
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 :domain
     skos:prefLabel "domain"@en;
-    skos:definition "A domain is the targeted subject area for a software-intensive system. It is the sphere of knowledge and activity around which the software's processes, rules, and data revolves"@en.
+    skos:definition ```
```

Comment:
My point about triple backticks was about the backticks not the triple. 
Single quote as in '. rather than double quote "

Unless you have reason to not use single quotes, then lets use single quotes for blocks.